bin/node/src/chain_spec.rs

```diff
@@ -272,7 +167,7 @@ pub fn testnet1_config(chain_params: ChainParams) -> Result<ChainSpec, String> {
         Some(
             [(
                 "tokenSymbol".to_string(),
-                serde_json::Value::String("TZERO".into()),
+                serde_json::Value::String("DZERO".into()),
```

Comment:
`DZERO` is supposed to be the devnet token symbol, did you change it here by mistake?

Reply:
there is one common `config` used by all "nets". The idea being that everything that we do not want hard-coded is passed as command line argument. Could add it to cli arguments or hard-code if we have a decision on what the token should be (AZERO?).

Reply:
It was `TZERO` for the testnet and `AZERO` for the mainnet, although you should probably confirm that last bit with Antek.

Reply:
ok, I will just make all of these parameters part of the CLI struct (ChainParams)